compiler/merge.cc

```diff
@@ -53,6 +53,71 @@ bool MaybeMergeSplitConcat(Graph* graph, Node* node) {
     return true;
 }
 
+bool MaybeMergePadConv(Graph* graph, Node* pad) {
+    if (pad->value() != 0.0 || pad->mode() != "constant") {
+        return false;
+    }
+
+    auto const& pads = pad->pads();
```

Comment:
`const std::vector<int64_t>&` please. This is my personal preference, but I'm not a big fun of `auto` for simple types.

Copy and paste from a previous comment in another review:

Personally, I'm not a big fan of auto. IMHO, it sometimes hurts readability significantly while it only saves you a few key strokes. I think I allow myself to use auto only for complex types. Typical examples are STL iterators, lambdas, and smart pointers.